### PR TITLE
Move overland roughness assignment out of time cycling disaggregation routine

### PR DESCRIPTION
**Pull request by aubreyd**
TYPE: enhancement

KEYWORDS: overland, parameters

SOURCE: Aubrey D, NCAR 

DESCRIPTION OF CHANGES:
The overland roughness parameter is being setup on the high-res grid in the disaggregation routine, but this routine is called every LSM timestep and the overland roughness parameter is not time-varying. It should be moved into the initialization routine.

ISSUE: The GitHub Issue that this PR addresses. For issue number 123, it would be:
```
Fixes #123
```

TESTS CONDUCTED:
Gives identical solution to pre-change on 1 core. DOES NOT CURRENTLY PASS NCORES... (help!).

### Checklist
 - [ ] Closes issue #xxxx (An issue must exist or be created to be closed. The issue describes and documents the problem and general solution, the PR describes the technical details of the solution.) 
 - [ ] Tests added (unit tests and/or regression/integration tests) NOT NEEDED.
 - [x] Backwards compatible
 - [ ] Requires new files? If so, how to generate them. NO NEW FILES REQUIRED.
 - [ ] Fully documented. INTERNAL CODE PERFORMANCE CHANGE ONLY - NO NEW FEATURES.
 - [ ] Short description in the Development section of `NEWS.md` NOT NEEDED.

### CODE REVIEW

**Comment by scrasmussen** [comment]
Just wanted to say that I've been able to reproduce that one core gives an identical solution and more than one core doesn't. I'm currently looking into the issue and will try to get it fixed as soon as possible! (First real deep dive into the code base so taking longer than expected)

File: trunk/NDHMS/Routing/module_RT.F [hunk 1]
```diff
@@ -1215,8 +1216,20 @@ subroutine LandRT_ini(did)
...
+     !Create overland roughness high-res grid
+     print *, "ADCHECK_pre ncoarse 2 2 nfine 11 53:", rt_domain(did)%OV_ROUGH2d(2, 2), rt_domain(did)%overland%properties%roughness(11,53)
+     call regrid_lowres_to_highres(did, rt_domain(did)%OV_ROUGH2d, rt_domain(did)%overland%properties%roughness, &
+                                    rt_domain(did)%ixrt, rt_domain(did)%jxrt)
+     print *, "ADCHECK_post_regrid nfine 2 2:", rt_domain(did)%overland%properties%roughness(2,2)
+     print *, "ADCHECK_post_regrid nfine 11 53:", rt_domain(did)%overland%properties%roughness(11,53)
+     !Apply scaling factor
+     rt_domain(did)%overland%properties%roughness = rt_domain(did)%overland%properties%roughness * rt_domain(did)%OVROUGHRTFAC
+     print *, "ADCHECK_post_fac nfine 2 2:", rt_domain(did)%overland%properties%roughness(2,2)
+     print *, "ADCHECK_post_fac nfine 11 53:", rt_domain(did)%overland%properties%roughness(11,53)
```
Comment (rcabell) [inline trunk/NDHMS/Routing/module_RT.F:1229]: Probably don't want to leave these `print *` statements in; either remove or move to HYDRO_D (or, even better, we should add a `SUBROUTINE DEBUG_PRINT()` facility and refactor all the HYDRO_D prints!)

**Review by rcabell — DISMISSED** [review]
Updated PR now passes CI testing on CONUS and Croton. Recommend merge.

**Review by scrasmussen — APPROVED** [review]
Removed some debug print statements and commented out debug code. 
Small changes, now everything looks good to me now! Nice work

**Review by rcabell — APPROVED** [review]
New updates ok by me

**Pull request merged**